cmd/snap/cmd_interfaces.go

```diff
@@ -100,19 +121,20 @@ func (x *cmdInterfaces) Execute(args []string) error {
 		}
 		// The OS snap is special and enable abbreviated
 		// display syntax on the slot-side of the connection.
-		if slot.Snap == "core" || slot.Snap == "ubuntu-core" {
+		if slot.Snap == "core" || slot.Snap == "ubuntu-core" || slot.Snap == snap.Nickname("core") {
```

Comment:
We have a growing number of places that do check like this, I wonder if it's time for a helper.

Reply:
AFAICT this is the only place where all 3 options appear.